tmdb/parsers/seasons.py

```diff
@@ -0,0 +1,32 @@
+"""Parsers of season objects."""
+
+from typing import List
+
+from ..datatypes import Season, Episode
+from .base import Parser
+
+
+class SeasonParser(Parser[Season]):
+
+    object_class = Season
+
+    @staticmethod
+    def _parse_episode(episode: dict) -> Episode:
+        number: int = episode['episode_number']
+        synopsis: str = episode['overview']
+        return Episode(number=number, synopsis=synopsis)
+
+    def _get_list_episodes(self, data: dict) -> List[Episode]:
+        episodes: List[dict] = data['episodes']
+        list_episodes = []
+        for episode in episodes:
+            list_episodes.append(self._parse_episode(episode))
```

Comment:
This is nice, but it clearly looks like we should use an `EpisodeParser` here too.

We may need to parse more detailed episode information in the future, but that's fine; we'll just have to create an `EpisodeParser(ParserGroup)` like we did for `ShowParser`. :smile: